README.md

```diff
@@ -1,5 +1,5 @@
 # Job Manager
-See also the [development guide](#development) below
+See also the [development guide](#development) below.
```

Comment:
This looks better. Didn't notice that the dot is blocking the anchor from working lol Thanks for the change here!